### ISSUE TITLE
Relay doesn't save all my stuff?

### ISSUE THREAD

**Issue by ogunden** [servuscms/servus#17]
I tried to migrate my data over from another relay, but found that servus is very picky about what kinds of events it saves. In particular, I get a lot of these:

```
{"level":30,"time":1729461853964,"msg":"Ignoring event of unknown kind: 6."}     
{"level":30,"time":1729461853976,"msg":"Ignoring event of unknown kind: 10002."} 
{"level":30,"time":1729461853976,"msg":"Ignoring event of unknown kind: 10013."}                         
{"level":30,"time":1729461853977,"msg":"Ignoring event of unknown kind: 10007."} 
{"level":30,"time":1729461853977,"msg":"Ignoring event of unknown kind: 10050."} 
{"level":30,"time":1729461853977,"msg":"Ignoring event of unknown kind: 7."}     
{"level":30,"time":1729461853978,"msg":"Ignoring event of unknown kind: 3."}
```

I was wanting to "own a copy of all my data", so I guess I was expecting servus to save this stuff. If the other relays I use decided to delete my stuff, I'd still have my own original notes but that's it, I guess.

**Comment by ibz** [comment]
You are right. Currently it only saves events it knows about and it will render as web pages (notes, long form content...). But it would be indeed good to simply save all events. I will look at that. Thank you for trying it out!

BTW, what tool did you use to try to migrate your data?

**Comment by ogunden** [comment]
I used migstr (https://github.com/tamstrup/migstr) -- super convenient!

**Comment by ibz** [comment]
Done. Saving all events now (in the last release) - even events of unknown kind. Makes sense, so one can be sure to have a complete archive.

`migstr` is a very good tool indeed, thank you!
